samples/excel/85-preview-apis/pivottable-pivotlayout.yaml

```
@@ -0,0 +1,322 @@
+              ["E Farms", "Apple", "Conventional", 245, 2200],
+              ["E Farms", "Kiwi", "Conventional", 200, ""],
+              ["F Farms", "Kiwi", "Organic", 100, ""],
+              ["F Farms", "Lemon", "Conventional", "", 270]
+            ];
+
+            const range = dataSheet.getRange("A1:E21");
+            range.values = data;
+            range.format.autofitColumns();
+
+            pivotSheet.activate();
+
+            const rangeToAnalyze = context.workbook.worksheets.getItem("Data").getRange("A1:E21");
+            const rangeToPlacePivot = context.workbook.worksheets.getItem("Pivot").getRange("A2");
+            const pivotTable = context.workbook.worksheets
+              .getItem("Pivot")
+              .pivotTables.add("Farm Sales", rangeToAnalyze, rangeToPlacePivot);
+
+            pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Farm"));
+            pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Type"));
+
+            pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold at Farm"));
+            pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold Wholesale"));
+
+            // Add the following if you'd like to see how the layout settings affect a column hierarchy.
+            //pivotTable.columnHierarchies.add(pivotTable.hierarchies.getItem("Classification"));
+
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to work with the PivotLayout to display the PivotTable.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+          <h3>Set up</h3>
+          <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Setup sample</span>
+            </button>
+        </section>
+        <section class="samples ms-font-m">
+          <h3>Try it out</h3>
+          <button id="change-layout" class="ms-Button">
+              <span class="ms-Button-label">Change layout type</span>
+          </button>
+          <button id="add-alt-text" class="ms-Button">
+              <span class="ms-Button-label">Add alt text</span>
+          </button>
+          <button id="add-line-spacing" class="ms-Button">
+              <span class="ms-Button-label">Add line spacing</span>
+          </button>
+          <button id="repeat-item-labels" class="ms-Button">
+              <span class="ms-Button-label">Repeat item labels</span>
+          </button>
+          <h4>Header and total settings</h4>
```

To keep this spacing around this H4 consistent with the spacing around other H4s in this snippet: 
```suggestion
          <br>
          <h4>Header and total settings</h4>
```